manager/src/main/java/io/gingersnapproject/search/opensearch/OpenSearchBackend.java

```diff
@@ -12,12 +15,14 @@
 import io.quarkus.arc.lookup.LookupIfProperty;
 import io.smallrye.mutiny.Uni;
 
-@LookupIfProperty(name = "service.opensearch.enabled", stringValue = "true")
+@LookupIfProperty(name = SearchBackend.PROPERTY, stringValue = "opensearch")
 @ApplicationScoped
 public class OpenSearchBackend implements SearchBackend {
 
    @Inject
-   private RestClient restClient;
+   RestClient restClient;
+
+   Set<String> mappedRules = ConcurrentHashMap.newKeySet();
 
    @Override
    public Uni<String> mapping(String indexName) {
```

Comment:
Not for this PR, but I guess we'll need a way to remove a mapping.

Reply:
Yeah, I figure that will come as part of the listener stuff. There are lot of issues there still that need to be ironed out. Removing map, index, clearing index etc